engine/tests/story/fabula/test_world_materialization.py

```
@@ -223,6 +223,31 @@ def test_materialize_block_creates_action_edges(mock_world):
+    from tangl.story.episode.block import Block
+    from tangl.vm.provision.open_edge import Dependency
+
+    template = BlockScript(
+        label="start",
+        obj_cls=Block,
+        text="Beginning",
+        actions=[{"text": "Continue", "successor": "next"}],
+    )
+
+    graph = StoryGraph(label="test", world=mock_world)
+    block = mock_world._materialize_from_template(template, graph)
```

Ruff (0.14.10)

Local variable `block` is assigned to but never used

Remove assignment to unused variable `block`

(F841)